### PR TITLE
Move automated analysis file upload to upload_analysis_files.py

### PR DESCRIPTION
**Pull request by as2388**
Copied from https://github.com/AfricasVoices/Project-WUSC-COVID19-ADAPTATION/pull/25

### CODE REVIEW

File: automated_analysis.py [hunk 1]
```diff
@@ -688,48 +682,12 @@ def set_survey_percentages(survey_counts, total_survey_counts):
...
-        drive_client_wrapper.update_or_create_batch(
-            paths_to_upload, f"{pipeline_configuration.drive_upload.analysis_graphs_dir}/maps/urban",
-            target_folder_is_shared_with_me=True, recursive=True
-        )
-    else:
-        log.info("Skipping uploading to Google Drive (because the pipeline configuration json does not contain the key "
-                 "'DriveUploadPaths')")
+        fig.write_image(f"{automated_analysis_output_dir}/graphs/{plan.raw_field}_by_gender_normalised.png", scale=IMG_SCALE_FACTOR)
+
+    log.info("Python script complete")
```
Comment (IsaackMwenda) [inline automated_analysis.py]:
```suggestion
    log.info("automated analysis python script complete")
```

**Review by IsaackMwenda — APPROVED** [review]

File: automated_analysis.py [hunk 2]
```diff
@@ -55,26 +55,20 @@
 
     messages_json_input_path = args.messages_json_input_path
     individuals_json_input_path = args.individuals_json_input_path
-    output_dir = args.output_dir
```
Comment (IsaackMwenda) [inline automated_analysis.py:58]: Rem to update docker-run-automated-analysis.sh to refer to AUTOMATED_ANALYSIS_OUTPUT_DIR

Comment (as2388) [reply automated_analysis.py:58]: Updated, and changed output-graphs/ to automated-analysis-outputs/ too, thanks

File: automated_analysis.py (hunk 1 above)
Comment (as2388) [reply automated_analysis.py]: Will revert this here and move to another PR so I can update the other scripts appropriately too.

**Pull request merged**